modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/PartitionReconciliation.java

```diff
@@ -222,20 +223,22 @@ private ReconciliationResult partitionReconciliationCheck(
 
                         break;
 
-                    case LOAD_FACTOR:
-                        strVal = argIter.nextArg("The load factor should be specified.");
+                    case PARALLELISM:
+                        strVal = argIter.nextArg("The parallelism level should be specified.");
 
                         try {
-                            loadFactor = Double.parseDouble(strVal);
+                            parallelism = Integer.parseInt(strVal);
                         }
                         catch (NumberFormatException e) {
-                            throw new IllegalArgumentException("Invalid load factor: " + strVal +
-                                ". Double value between 0 (exclusive) and 1 (inclusive) should be used.");
+                            throw new IllegalArgumentException("Invalid parallelism: " + strVal +
+                                ". Integer value from 1 to 128 should be specified, " +
+                                "or 0 (Runtime.getRuntime().availableProcessors() will be used in such case).");
                         }
 
-                        if (loadFactor <= 0 || loadFactor > 1) {
-                            throw new IllegalArgumentException("Invalid load factor: " + strVal +
-                                ". Double value between 0 (exclusive) and 1 (inclusive) should be used.");
+                        if (parallelism < 0 || parallelism > 128) {
```

Comment:
You should extract this text and above to var.

Reply:
Done